packages/cli/test/integration/generators/copyright-monorepo.integration.js

```diff
@@ -38,7 +38,12 @@ describe('lb4 copyright for monorepo', function () {
           additionalFiles: SANDBOX_FILES,
         }),
       )
-      .withOptions({gitOnly: false, owner: 'ACME Inc.', license: 'MIT'});
+      .withOptions({
+        gitOnly: false,
+        owner: 'ACME Inc.',
+        license: 'MIT',
+        packageManager: 'npm', // Default
```

Comment:
What's going to happen if this option is not set? Does it default to `npm`?

Reply:
I do not know what is happening... The other tests take the default value that is NPM but those do not.
```
➜  loopback-next git:(changes-in-cli) ./packages/cli/bin/cli-main.js app -h
Usage:
  lb4 app [<name>] [options]

Options:
  -h,    --help             # Print the generator's options and usage
         --skip-cache       # Do not remember prompt answers                                Default: false
         --skip-install     # Do not automatically install dependencies                     Default: false
         --force-install    # Fail on install dependencies error                            Default: false
         --ask-answered     # Show prompts for already configured options                   Default: false
         --applicationName  # Application class name
         --docker           # Include Dockerfile and .dockerignore
         --repositories     # Include repository imports and RepositoryMixin
         --services         # Include service-proxy imports and ServiceMixin
         --apiconnect       # Include ApiConnectComponent
         --description      # Description for the application
         --outdir           # Project root directory for the application
         --eslint           # Enable eslint
         --prettier         # Enable prettier
         --mocha            # Enable mocha
         --loopbackBuild    # Use @loopback/build
         --vscode           # Use preconfigured VSCode settings
         --private          # Mark the project private (excluded from npm publish)
  -c,    --config           # JSON file name or value to configure options
  -y,    --yes              # Skip all confirmation prompts with default or provided value
         --format           # Format generated code using npm run lint:fix
  -pm,   --packageManager   # Change the default package manager                            Default: npm

Arguments:
  name  # Project name for the application  Type: String  Required: false

➜  loopback-next git:(changes-in-cli) 
```

Reply:
I was able to fix the issue and pushed a commit back to your branch. Please note it's also rebased against latest master. You will need to reset your local checkout.

Reply:
Done. Thanks for helping me